lib/main.dart

```diff
@@ -13,103 +14,319 @@ class MyApp extends StatelessWidget {
+      backgroundColor: const Color.fromRGBO(32, 30, 33, 1),
       appBar: AppBar(
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
-      ),
-      body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug painting" (press "p" in the console, choose the
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+          backgroundColor: const Color.fromRGBO(32, 30, 33, 1),
+          title: Container(
+            margin: EdgeInsets.only(top: 8, left: leftMargin),
+            child: const Text(
+              'Home',
+              style: TextStyle(
+                  fontSize: 18,
+                  fontWeight: FontWeight.w100, // light
+                  color: Color.fromRGBO(227, 225, 228, 1)),
             ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+          ),
+          actions: [
+            IconButton(
+              onPressed: () {},
+              icon: Image.asset('assets/upload_icon.png'),
+              iconSize: 30,
+              color: Colors.grey,
             ),
-          ],
-        ),
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.notifications_none),
+              iconSize: 30,
+              color: Colors.grey,
+            ),
+          ]),
+      body: Stack(children: [SingleChildScrollView(
+
+        child: Column(children: [
+          Container(
+            color: const Color.fromRGBO(23, 21, 24, 1),
+            height: 15,
+          ),
+          _relatedSongsList(context),
+          Container(
+            color: const Color.fromRGBO(23, 21, 24, 1),
+            height: 15,
+          ),
+          _chartSongsList(context),
+          Container(
+            color: const Color.fromRGBO(23, 21, 24, 1),
```

Comment:
I recommend storing appearance and metrics in constants.dart file

Advantages:
1. In the future whenever you want to change your custom widgets or resources constants, you have to change it in only one place and it will reflect in all places.
2. Views code looks more clean and readable.